### PR TITLE
Chore: Auth Context

### PR DESCRIPTION
**Pull request by mr3nz1**
### What does this pull request do?
This PR provides a global way to keep track of whether the user is logged in or not

### Description
It keeps track of the following

- [x] Auth token
- [x] Refresh token
- [x] user logged in

### CODE REVIEW

**Review by Irirwanirira — APPROVED** [review]
Good job

File: app/(app)/ActionMenu/index.tsx [hunk 1]
```diff
@@ -16,8 +16,24 @@ import { SvgXml } from "react-native-svg";
 import {NotificationIcon,NotificationIconDark} from "@/assets/icons/Profile/Icons";
 import { ThemeContext } from "@/ctx/ThemeContext";
 import { blackHeart, whiteHeart } from "@/components/UI/icons/blackHeart";
+import { Session } from "@supabase/supabase-js";
+import { supabase } from "@/lib/supabase";
 
 export default function Index() {
+  import {} useConte
```
Comment (KeynesBizimana) [inline app/(app)/ActionMenu/index.tsx]: consider fixing this ,import should be used at the beginning

**Review by KeynesBizimana — CHANGES_REQUESTED** [review]

**Review by KeynesBizimana — APPROVED** [review]

File: app/(auth)/SignIn&SignOut/SignInBlankForm.tsx [hunk 2]
```diff
@@ -79,18 +81,27 @@ const Login = () => {
...
+  //   password: password,
+    
+  // }) 
+  // if(error){
+  //   Alert.alert(error.message)
+  //   setLoading(false)
+  // }else{
+  //   await router.push('/(app)/ActionMenu');
+  //   setLoading(false);
+  // }
```
Comment (Nkbtemmy) [inline app/(auth)/SignIn&SignOut/SignInBlankForm.tsx]: Don't push commented codes

File: ctx/AuthContext.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ 
+  async function login(email: string, password: string) {
+    try {
+      setLoading(true);
+      const { error } = await supabase.auth.signInWithPassword({
+        email,
+        password,
+      });
+      if (error) {
+        // Alert.alert(error.message);
```
Comment (Nkbtemmy) [inline ctx/AuthContext.tsx]: Remove even this small comments

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**